packages/dataviews/src/components/dataform/stories/index.story.tsx

```diff
@@ -113,6 +113,11 @@ const fields = [
 		label: 'Sticky',
 		type: 'boolean',
 	},
+	{
+		id: 'can_comment',
+		label: 'Allow people to leave a comment',
+		type: 'checkbox' as const,
```

Comment:
I think "checkbox" is not a type, it should be a control. 

So here you should be able to do

```
type: 'boolean',
Edit: 'checkbox'
```

Reply:
Oh yeah, you're right! I just thought every control had to have a matching type 😓